src/api/db/models/Image.ts

```diff
@@ -427,25 +462,6 @@ export class ImageModel {
         },
         { retries: 2 },
       );
-    };
-
-    try {
-      // find image, create label record
-      const project = await ProjectModel.queryById(context.user['curr_project']);
-
-      for (let oid = 0; oid < input.objects.length; oid++) {
-        const image = await ImageModel.queryById(input.objects[oid].imageId, context);
-
-        for (let lid = 0; lid < (input.objects[oid].labels || []).length; lid++) {
-          input.objects[oid].labels[lid] = reviewerLabelRecord(
-            project,
-            image,
-            input.objects[oid].labels[lid],
-          );
```

Comment:
I believe this was a bug.  It should have been `input.objects[oid].object.labels[lid]` as per the types:

https://github.com/tnc-ca-geo/animl-api/blob/2c66fa1413d35fa6710486019548b64f2364efbc/src/api/type-defs/inputs/CreateObjectsInput.ts#L1-L17